vendor/vendor.json

```diff
@@ -114,6 +114,7 @@
 		{"path":"github.com/golang/protobuf/ptypes","checksumSHA1":"/s0InJhSrxhTpqw5FIKgSMknCfM=","revision":"b4deda0973fb4c70b50d226b1af49f3da59f5265","revisionTime":"2018-04-30T18:52:41Z","version":"v1","versionExact":"v1.1.0"},
 		{"path":"github.com/golang/protobuf/ptypes/any","checksumSHA1":"3eqU9o+NMZSLM/coY5WDq7C1uKg=","revision":"b4deda0973fb4c70b50d226b1af49f3da59f5265","revisionTime":"2018-04-30T18:52:41Z","version":"v1","versionExact":"v1.1.0"},
 		{"path":"github.com/golang/protobuf/ptypes/duration","checksumSHA1":"ZIF0rnVzNLluFPqUebtJrVonMr4=","revision":"b4deda0973fb4c70b50d226b1af49f3da59f5265","revisionTime":"2018-04-30T18:52:41Z","version":"v1","versionExact":"v1.1.0"},
+		{"path":"github.com/golang/protobuf/ptypes/empty","checksumSHA1":"7Az4Zl9T11I+xOfjgs/3/YMJ24I=","revision":"89a0c16f4dc2a70c0ed864d8ef64878f24fdaa51","revisionTime":"2018-08-04T22:03:15Z"},
```

Comment:
We should probably make sure that this revision matches the other `ptypes` package. `b4deda0973fb4c70b50d226b1af49f3da59f5265` is the latest tag, so I think we should pin it there.

Reply:
Good idea, will do.